composer.json

```diff
@@ -19,7 +19,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.3 || ~8.0.0"
+        "php": "^7.3 || ^8.0.0"
```

Comment:
This should be `^7.3 || ~8.0.0 || ~8.1.0`.

We still don't know anything about PHP 8.2.0.